middleware/protect.js

```diff
@@ -17,7 +17,7 @@
 
 const UUID = require('./../uuid');
 
-function forceLogin (keycloak, request, response) {
+function forceLogin (keycloak, request, response, routeName) {
```

Comment:
I think we should attach the Keycloak config to the request. sth like
```javascript
req = {
 ...req,
 keycloakConfig: {
   action: "register" | "login" 
 }
}
```

Reply:
if (keycloak.redirectToLogin(request)) {
      // keycloak config will be added here in the request object
      forceLogin(keycloak, request, response, routeName);
    }